v3/packages.json

```
@@ -3734,6 +3881,47 @@
         }
       ]
     },
+    {
+      "id": "karoterra/confirmclose",
+      "name": "改変版Auls終了確認",
+      "overview": "プロジェクトを閉じる前に確認メッセージを表示する",
+      "description": "拡張編集に未保存のデータがある場合、プロジェクトを閉じる前に確認メッセージを表示します。Auls( http://auls.client.jp/ )さんの終了確認プラグインを勝手に改変したものです。",
+      "developer": "karoterra",
+      "originalDeveloper": "yu_noimage_",
+      "dependencies": ["exedit0.92"],
+      "pageURL": "https://scrapbox.io/karoterra/%E6%94%B9%E5%A4%89%E7%89%88Auls%E7%B5%82%E4%BA%86%E7%A2%BA%E8%AA%8D",
+      "downloadURLs": [
+        "https://scrapbox.io/karoterra/%E6%94%B9%E5%A4%89%E7%89%88Auls%E7%B5%82%E4%BA%86%E7%A2%BA%E8%AA%8D"
+      ],
+      "latestVersion": "v1.2",
+      "files": [{ "filename": "plugins/auls_confirmclose.auf" }],
```

```suggestion
      "files": [{ "filename": "auls_confirmclose.auf" }],
```
The original version should be placed in the same directory as `aviutl.exe`, so we should do the same.